src/rendering/RenderUtil.cc

```diff
@@ -2272,6 +2349,107 @@ void RenderUtil::ViewWireframes(const Entity &_entity)
     }
   }
 
+  return links;
+}
+
+/////////////////////////////////////////////////
+void RenderUtil::HideWireboxes(const Entity &_entity)
+{
+  if (this->dataPtr->wireBoxes.find(_entity)
+        != this->dataPtr->wireBoxes.end())
+  {
+    ignition::rendering::WireBoxPtr wireBox =
+      this->dataPtr->wireBoxes[_entity];
+    auto visParent = wireBox->Parent();
+    if (visParent)
+      visParent->SetVisible(false);
+  }
+}
+
+/////////////////////////////////////////////////
+void RenderUtil::ViewTransparent(const Entity &_entity)
+{
+  std::vector<Entity> visEntities;
+
+  if (this->dataPtr->linkToVisualEntities.find(_entity) !=
+      this->dataPtr->linkToVisualEntities.end())
+  {
+    visEntities = this->dataPtr->linkToVisualEntities[_entity];
+  }
+
+  // Find all existing child links for this entity
+  std::vector<Entity> links = this->FindChildLinks(_entity);
```

Comment:
how about `std::vector<Entity> links = std::move(this->FindChildLinks(_entity));` to avoid copying the vector

Reply:
[`8818c28`](https://github.com/ignitionrobotics/ign-gazebo/pull/878/commits/8818c2829b49b1545edc58cfc303b378974ce478)